Tests/SuperLinq.Test/TrySingleTest.cs

```diff
@@ -110,4 +110,15 @@ public void TrySingleEnumeratesOnceOnlyAndDisposes(int numberOfElements, string
 		var (cardinality, _) = seq.TrySingle("zero", "one", "many");
 		Assert.Equal(expectedCardinality, cardinality);
 	}
+
+	[Theory]
+	[InlineData(0, 0)]
+	[InlineData(1, 1)]
+	[InlineData(2, 0)]
```

Comment:
Code coverage pointed out that we're not testing collection branch. Please make a copy of `GetSequences` that returns a list of sequences and expected results. This way we can test both branches, for 0, 1, many.